src/runtime/crt/host/Makefile

```diff
@@ -38,7 +42,7 @@ CRT_LIBS = $(patsubst %, $(BUILD_DIR)/crt/lib%.a, $(CRT_LIB_NAMES))
 CRT_INCLUDES = $(glob crt/include/**)
 
 $(BUILD_DIR)/crt/lib%.a: $(glob crt/src/runtime/%/*.c)
-	${QUIET}cd crt && $(MAKE) \
+	${QUIET}cd crt && $(MAKE) -s \
```

Comment:
do you need this and verbose?

Reply:
yeah, otherwise you get messages like `make[1]: Entering directory '/tmp/tmptwf17bxw/generated-project/crt'`